pkg/k8sutil/status.go

```diff
@@ -254,3 +256,62 @@ func UpdateRollingUpgradeState(c client.Client, cluster *v1beta1.KafkaCluster, t
 	logger.Info("Rolling upgrade status updated", "status", timeStamp)
 	return nil
 }
+
+func UpdateListenerStatuses(c client.Client, cluster *v1beta1.KafkaCluster, logger logr.Logger, extListenerStatuses map[string]v1beta1.ListenerStatus) error {
+	typeMeta := cluster.TypeMeta
+
+	intListenerStatuses := createInternalListenerStatuses(cluster)
```

Comment:
Inject `intListenerStatuses` as a parameter instead of computing inside this method.

Reply:
Fixed